pylxd/models/_model.py

```diff
@@ -110,6 +123,14 @@ def __init__(self, client, **kwargs):
             try:
                 setattr(self, key, val)
             except AttributeError:
+                global _seen_attribute_warnings
+                env = os.environ.get('PYLXD_WARNINGS', '').lower()
+                item = "{}.{}".format(self.__class__.__name__, key)
+                if env != 'always' and item in _seen_attribute_warnings:
+                    continue
+                _seen_attribute_warnings.add(item)
+                if env == 'none':
+                    continue
```

Comment:
Why bother adding it to the set if we're going to ignore it anyways? I'd think that it's a (tiny) waste of memory and would prefer:

```python
if env == 'none':
    continue
_seen_attribute_warnings.add(item)
```

Reply:
Yes, I went back and forth on this; for consistency always adding the item made sense.  The memory issue was largely irrelevant, although I did consider it.   I can change it if you like, but it doesn't change the functionality except for it the env var is changed.  I went for "changing the env var would only result in new warnings from that point" rather than "changing the env var makes it start working, but doing it again wouldn't make any difference".

Reply:
Can you tell me how to change the environment variable of a running process? It seems to me that the only way to update that config would be to restart the process anyways, which means it looses any "saved state?"

Reply:
Doh; indeed.  ENV vars are fixed for the run-time of the program.  I still have a nagging feeling it's more correct, from the perspective of how the variable is used; it changes the meaning of the variable.  If later, it's decided (say) that a method can be used to modify this, then, unless this is then changed, it would become a bug.

Reply:
Given the probably small number of unknown attributes, this doesn't bother me much